src/modules/contact.py

```diff
@@ -35,7 +38,21 @@ def add_birthday(self, birthday):
         except ValueError as e:
             print(e)
 
+    def add_email(self, email):
+        try:
+            self.email = Email(email)
+        except ValueError as e:
+            print(e)
+
+    def add_address(self, address):
+        try:
+            self.address = Address(address)
+        except ValueError as e:
+            print(e)
+
     def __str__(self):
         return (f"Contact name: {self.name.value},"
                 f" phones: {'; '.join(p.value for p in self.phones)},"
-                f" birthday: {self.birthday}")
+                f" birthday: {self.birthday}"
+                f" email: {self.email}"
+                f" address: {self.address}")
```

Comment:
This will throw an error if Contact does not have self.email or self.address

Reply:
Discussed this case in Slack. It was not error with function, it is problem with local storage file.